WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/Preview/ProductDetailPreviewView.swift

```diff
@@ -0,0 +1,252 @@
+        let content: String?
+        let isLoading: Bool
+        let dummyText = ProductDetailPreviewView.Constants.dummyText
+
+        typealias Layout = ProductDetailPreviewView.Layout
+
+        var body: some View {
+            Text(content ?? dummyText)
+                .bodyStyle()
+                .multilineTextAlignment(.leading)
+                .padding(Layout.contentPadding)
+                .frame(maxWidth: .infinity, alignment: .leading)
+                .background(Color(.init(light: .systemGray6, dark: .tertiarySystemBackground)))
+                .cornerRadius(Layout.cornerRadius)
+                .redacted(reason: isLoading ? .placeholder : [])
+                .shimmering(active: isLoading)
+        }
+    }
+
+    /// View to contain product details with title, value and image.
+    struct TitleAndValueDetailRow: View {
+        let title: String
+        let value: String?
+        let image: UIImage
+        let isLoading: Bool
+        var cornerRadius: CGFloat = 0
+        let dummyText = ProductDetailPreviewView.Constants.dummyText
+
+        typealias Layout = ProductDetailPreviewView.Layout
+
+        var body: some View {
+            HStack(alignment: .top, spacing: Layout.contentPadding) {
+                Image(uiImage: image)
+                    .renderingMode(.template)
+                    .foregroundColor(.secondary)
+                VStack(alignment: .leading, spacing: Layout.detailVerticalSpacing) {
+                    Text(title)
+                        .bodyStyle()
+                    Text(value ?? dummyText)
+                        .subheadlineStyle()
+                        .multilineTextAlignment(.leading)
+                }
+            }
+            .padding(Layout.contentPadding)
+            .frame(maxWidth: .infinity, alignment: .leading)
+            .background(Color(.init(light: .systemGray6, dark: .tertiarySystemBackground)))
+            .cornerRadius(cornerRadius)
+            .redacted(reason: isLoading ? .placeholder : [])
+            .shimmering(active: isLoading)
+        }
+    }
+}
+
+fileprivate extension ProductDetailPreviewView {
+    enum Layout {
+        static let insets: EdgeInsets = .init(top: 24, leading: 16, bottom: 16, trailing: 16)
+        static let titleBlockBottomSpacing: CGFloat = 24
+        static let blockVerticalSpacing: CGFloat = 24
+        static let contentVerticalSpacing: CGFloat = 8
+        static let contentPadding: CGFloat = 16
+        static let cornerRadius: CGFloat = 8
+        static let detailVerticalSpacing: CGFloat = 4
+    }
+    enum Constants {
+        static let dummyText = "Lorem ipsum dolor sit amet, consectetur adipiscing elit"
+    }
+    enum Localization {
+        static let title = NSLocalizedString(
+            "Preview",
+            comment: "Title on the add product with AI Preview screen."
+        )
+        static let subtitle = NSLocalizedString(
+            "Don't worry. You can always change those details later.",
```

Comment:
> "Don't worry. You can always change those details later."

Opinion: I think changing "those" into "below"/"these" will add more clarity.

Reply:
I was unsure about this wording too. Asking Ann in the design, will update this later 👍

Reply:
I've changed it to "below" as it surely gains more clarity. :) Thanks!

Reply:
Updated in 238a341447d96be21005e962f45493e915053404.